subsys/net/l2/openthread/Kconfig

```diff
@@ -279,6 +279,13 @@ config OPENTHREAD_MAC_SOFTWARE_RETRANSMIT_ENABLE
 	help
 	  Set y if the radio supports tx retry logic with collision avoidance (CSMA)
 
+config OPENTHREAD_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE
+	bool "Enable software CSMA backoff logic"
+	default y
+	help
+	  Set y to enable software CSMA backoff. The option can be disabled if
+	  the radio have hardware support for this feature (IEEE802154_HW_CSMA).
```

Comment:
```suggestion
	  the radio has hardware support for this feature (IEEE802154_HW_CSMA).
```

Reply:
Ehh, I had this fixed but didn't save before push :) Will fix.